src/shared/apis/queryCache.ts

```diff
@@ -139,6 +139,10 @@ class QueryCache {
     this.cache = {}
   }
 
+  getById = (queryID: string) => {
```

Comment:
We should remove this since it’s not valid and I think isn’t being used

Reply:
done.